assets/bulk-add.js

```diff
@@ -0,0 +1,37 @@
+class BulkAdd extends HTMLElement {
```

Comment:
I will be re-using some of this functionality for this: https://github.com/orgs/Shopify/projects/6397/views/28?pane=issue&itemId=59790024

So I am creating an extendable component I will also use for Quick Add